### ISSUE TITLE
Upgrade project to use Fast.AI V1.0 and Pytorch 1.0

### ISSUE THREAD

**Issue by jantic** [jantic/DeOldify#12]
Will probably wait until Pytorch 1.0 is in stable release.

**Comment by avaika** [comment]
Please also add requirements.txt to the prohect.

**Comment by jantic** [comment]
> Hi!
> 
> Have you updated dependencies requirements in the README ?

They're indeed in the readme and updated, but not terribly user friend.  I put this project up just a few days ago not thinking I have -this- much attention so I wasn't too worried about user friendliness for thousands of people.  Obviously the situation has changed (LOL).

I think i'm going to prioritize this soon:  https://github.com/jantic/DeOldify/issues/22 which covers requirements.txt.  I was originally relying on users to just install fast.ai which basically takes care of all the dependencies and then just plop this code on top of it.  But yeah, now that there's thousands of eyeballs on this, that just doesn't work anymore.

**Comment by jantic** [comment]
Update:  Turns out that Fast.ai 1.0 is still quite unstable.  The conda install, for example, is clearly from at least a month ago and is missing quite a few key things from the current source.  There's still a lot of refactoring going on in the code base there.  

Thus, I'm going to hold off on this upgrade indefinitely.  I may wind up not doing it altogether because it may just not be worth the hassle based on what I saw.  That isn't to say there isn't benefits (augmentations and transformations should be way faster, for example), but it's looking like for this particular project the benefits may not justify how much would have to be rewritten to get it to work. 

Keep in mind that this doesn't preclude the project from being upgrade to Pytorch 1.0. Now I'll wait until that's in stable release mode before I get the project on using that.  

What I've done instead is made the install of this project, via Conda, super easy (see readme for instructions).  I achieved this partially by just forking Fast.ai 0.7 and making it a part of the project.  This will also shield the project from whatever the fate of Fast.ai 0.7 is and we simply won't have to worry about it.
